templates/partials/person_table.html

```diff
@@ -40,10 +41,13 @@
           {% endif %}
         {% endif %}
       </td>
+      <td class="text-right">
+        {{ salary.job.salaries.first().extra_pay|format_salary }}
```

Comment:
Ditto here, except `salary.extra_pay`.

Reply:
IIRC, we'll want to display the raw extra pay figure here, instead of something like `2k`.

Reply:
Do you mean to omit the `format_salary` filter for `extra_pay` and `salary_at_hand`? The filter isn't abbreviating the numbers that much, just rounding the cents. @hancush

Reply:
Yes, that's what I mean. Let's keep full precision for now, since we aren't formatting base pay.